azuredeploy.json

```diff
@@ -163,14 +163,42 @@
       "metadata": {
         "description": "The URL of resources, such as templates, scripts, and DSC modules, that the ARM deployment template depends on"
       },
-      "defaultValue": "https://raw.githubusercontent.com/teradici/deploy/master"
```

Comment:
Need to back this out before commit. (And I don't think we need to have a default here at all actually since deploy-CAM.ps1 always passes a value.)

Reply:
Will fix.